HCTextField/Source/HCTextField.swift

```diff
@@ -101,27 +116,27 @@ open class HCTextField: UITextField {
 
         if checkType.contains(.email) && !isEmail(text) {
 
-            passedCheck = false
+            HCTextField.checkResults[self] = false
```

Comment:
sry, this is not needed. We are going to use `HCTextFieldManager` class to manage all the textfields.